src/Sylius/Behat/Context/Api/Shop/CheckoutContext.php

```diff
@@ -716,11 +722,14 @@ public function thereShouldBeNoTaxesCharged(): void
     }
 
     /**
-     * @Then my order's locale should be :localeCode
+     * @Then my order's locale should be :locale
      */
-    public function myOrderLocaleShouldBe(string $localeCode): void
+    public function myOrderLocaleShouldBe(LocaleInterface $locale): void
     {
-        Assert::same($this->responseChecker->getValue($this->ordersClient->getLastResponse(), 'localeCode'), $localeCode);
+        Assert::same($this
+            ->responseChecker
+            ->getValue($this->ordersClient->getLastResponse(), 'localeCode'), $locale->getCode()
+        );
```

Comment:
```suggestion
        Assert::same($this->responseChecker->getValue(
            $this->ordersClient->getLastResponse(), 'localeCode'),
            $locale->getCode()
        );
```

I feel that this is more elegant and better to read, but I might be wrong 😄

Reply:
We should 
```suggestion
        Assert::same(
            $this->responseChecker->getValue(
                $this->ordersClient->getLastResponse(), 'localeCode'
            ),
            $locale->getCode()
        );
```

Possibly we should change it that, your proposition is ok but in first line we have opened two parenthesis and should be closed in the same next lines. In your case second parenthesis is closed in second line and first parenthesis is closed in fourth line. We should remember about this rules 🚀